src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -1176,6 +1176,18 @@ void GetPlantInput(EnergyPlusData &state)
                         }
                         break;
                     }
+                    case PlantEquipmentType::HeatPumpFuelFiredHeating: {
+                        this_comp.compPtr = EIRPlantLoopHeatPumps::EIRFuelFiredHeatPump::factory(
+                            state, PlantEquipmentType::HeatPumpFuelFiredHeating, CompNames(CompNum));
+                        this_comp.CurOpSchemeType = OpScheme::Invalid;
+                        break;
```

Comment:
I think our convention is to put the `break` outside of the `case` scoping braces as you are doing below. I'm not sure if anyone cares or not.

Reply:
Looked at this earlier but would like to just keep all of these as it is following the patterns for all other plant equipment for now. It does not seems to be making any differences or causing any ambiguity. So this is kept as is.